### DISCUSSION TITLE
Just got Business plan, but custom branding seems to be incomplete

### DISCUSSION THREAD

**Discussion by dannyshin76** [Budibase/budibase#12379] [Help] [answered]
One of the major purposes of deciding to get Business plan was the custom branding. However, after getting the Business plan and using all the custom branding settings, I'm still finding "Budibase" in open graph properties:
![image](https://github.com/Budibase/budibase/assets/93232582/feb28d74-0541-437e-9a1d-7e54ed07187a)

Could anyone let me know how to replace those properties with my own website name?
(I am using Budibase self-hosting v2.11.36)

**Comment by ConorWebb96** [comment, marked as answer]
Hey @dannyshin76,

These don't do much of anything, I've created an issue [here](https://github.com/Budibase/budibase/issues/12399). When this is completed these will likely be removed completely or for the site name use the organisation name instead. 

Hopefully this resolves your issue.
